### PR TITLE
Hunter Holland Sprint Review 2

### PR DESCRIPTION
**Pull request by whathunterbuilds**
At this point, it is hard to see the exact vision the CoachPlan group has in mind. Functionality feels limited at this point. How will a user interact with this website? As a coach will you be able to drag and drop training routines to their athletes? I feel like more time was spent on getting Firebase set up and working than getting something that is minimally viable. That being said, the Firebase implementation is absolutely awesome! It is so seamless and it requires no set up for people looking at this code. During this next sprint, I will definitely be spending time getting our database set up in the cloud to have a similar user experience. It really is very convenient. The program was available. There are not very many comments throughout the code, however, it is easy to understand because most of the application has been covered throughout the course of our lectures. The program does compile.

### CODE REVIEW

**Comment by whathunterbuilds** [comment]
The greatest learning tool in this application was firebase. I learned how to initialize it using a JSON file. This JSON file had all of the configuration necessary to use the Firebase this group created. I have never used FileInputStream personally, so that was also a good learning experience. 
```
FileInputStream serviceAccount =
				  new FileInputStream("./serviceAccount.json");
```
So as mentioned previously, this JSON has all the information Firebase needs to initialize. Finally, you can use that FileStream to pull credentials when you are building Firebase.
```
FirebaseOptions options = new FirebaseOptions.Builder()
				  .setCredentials(GoogleCredentials.fromStream(serviceAccount))
				  .setDatabaseUrl("https://coachplan-bcb7e.firebaseio.com")
				  .build();
```

**Comment by whathunterbuilds** [comment]
Code Justifications: 
The first thing I noticed was the home button in the navbar. I removed that button and made the CoachPlan logo the new home button. I feel like this will result in a more familiar experience for users. Next, I found that if I typed in my login information, I would be sent back to the index page. I changed that functionality, so that it would not remove me from where I was. I think keeping the old way would have been frustrating for users. Finally, I got rid of all alerts that popped whether you were logging in or out or whatever else. I would recommend using a banner message instead of alerts. Alerts don't look very good, and they add an unnecessary click. The user should just expect to be logged in when their credentials are correct.

File: src/main/resources/static/index.js [hunk 1]
```diff
@@ -22,16 +22,13 @@ function login() {
 			console.log(currentUserId);
 			
 			if(currentUserId == 1 && currentUserPass == userPass) {
-				window.alert("Signed In");
```
Comment (discospiff) [inline src/main/resources/static/index.js:25]: Over-alerting can annoy users.  Good to remove these.

File: src/main/resources/templates/athlete.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 <body>
 	 <nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <a class="navbar-brand" href="#">CoachPlan</a>
+  <a class="navbar-brand" href="index">CoachPlan</a>
```
Comment (discospiff) [inline src/main/resources/templates/athlete.html:21]: This makes sense.

File: src/main/resources/templates/athleteList.html [hunk 3]
```diff
@@ -29,12 +29,11 @@
   </div>
 </nav>
 	<h1>Coach login Page</h1>
-	<!--
-	<p th:text="${'Welcome, ' + coachDTO + '!'}" />
```
Comment (discospiff) [inline src/main/resources/templates/athleteList.html:33]: I'd remove this entirely.  Commented-out code should not be committed to version control.  1)  You can always go back in history to see what used to be there.  2) If you're trying an experimental feature, you can do it in a branch, and not impact master.

File: src/main/resources/templates/index.html [hunk 4]
```diff
@@ -109,13 +109,12 @@
 
 
  <nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <a class="navbar-brand" href="#">CoachPlan</a>
+  <a class="navbar-brand" href="#" onclick="home()">CoachPlan</a>
```
Comment (discospiff) [inline src/main/resources/templates/index.html:112]: This is intuitive.

**Comment by discospiff** [comment]
Several good suggestions that reduce the technical debt of the program.  I recommend merging these changes.

**Comment by berthedn** [comment]
Thank you, I made the changes for the navbar and the login form. Have a nice day.

**Pull request closed without merging**